.github/actions/report-job-status/README.md

```diff
@@ -0,0 +1,31 @@
+# Send VK Teams notification on failure
+
+This action uses [report-job-status action](https://github.com/tarantool/actions/tree/master/report-job-status)
```

Comment:
I suggest using this variant:

`This action selects the corresponding chat for the notification on failure and uses [report-job-status action](https://github.com/tarantool/actions/tree/master/report-job-status) to send it.`

Reply:
Looks like fixed.